packages/grafana/go-mysql-server-sql-types-strings.go.patch

```diff
@@ -0,0 +1,19 @@
+--- ./vendor/github.com/dolthub/go-mysql-server/sql/types/strings.go.orig	2025-05-21 05:20:36.774867843 +0000
++++ ./vendor/github.com/dolthub/go-mysql-server/sql/types/strings.go	2025-05-21 05:21:43.995602742 +0000
+@@ -17,6 +17,7 @@
+ import (
+ 	"bytes"
+ 	"fmt"
++	"math"
+ 	"reflect"
+ 	"strconv"
+ 	strings2 "strings"
+@@ -41,7 +42,7 @@
+ 	TinyTextBlobMax   = charBinaryMax
+ 	TextBlobMax       = varcharVarbinaryMax
+ 	MediumTextBlobMax = 16_777_215
+-	LongTextBlobMax   = int64(4_294_967_295)
++	LongTextBlobMax   = int64(math.MaxInt32)
```

Comment:
@twaik There is one non-"type change fix" over here. This constant is used in grafana source only for passing to [`scanner.Buffer`](https://pkg.go.dev/bufio#Scanner.Buffer)

This function for some reason takes int instead of uint, so this hack is required for 32-bit targets

Reply:
Is there any way to define `math.MaxInt32 * 2` here for 64bit targets?